src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -362,8 +362,10 @@ class ManageApplicationContainer extends React.Component<
           label={CONSTANTS.RESUME_REQUEST_LABEL}
           mode={this.state.mode}
           hackerId={this.state.hackerDetails.id}
-          required={this.props.mode === ManageApplicationModes.CREATE}
-          value={fp.values.resume}
+          // required={this.props.mode === ManageApplicationModes.CREATE}
```

Comment:
Why was this commented?

Reply:
whoops sorry i really didn't clean up my code this morning, ignore it! will take it out